src/ui/zcl_abapgit_gui_page_main.clas.abap

```diff
@@ -177,7 +163,7 @@ CLASS zcl_abapgit_gui_page_main IMPLEMENTATION.
 
         mv_repo_key = lv_key.
         CREATE OBJECT ei_page TYPE zcl_abapgit_gui_page_view_repo
-        EXPORTING iv_key = lv_key.
```

Comment:
@larshp this was not caught by abaplint 
https://github.com/abaplint/abaplint/issues/1028

Reply:
see comment in abaplint/abaplint#1028